net-misc/feather/feather-1.0.1.ebuild

```
@@ -0,0 +1,61 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="A free, open-source Monero wallet"
+HOMEPAGE="https://featherwallet.org"
+SRC_URI="https://featherwallet.org/files/releases/linux/${PN}-${PV}-linux.zip -> ${P}.zip"
+
+# Feather is released under the terms of the BSD license, but it vendors
+# code from Monero and Tor too.
+LICENSE="BSD MIT"
+SLOT="0"
+KEYWORDS=""
```

Since this binary only works for amd64 linux, we could keyword that.
```suggestion
KEYWORDS="-* ~amd64"
```